dynamic-plugins/wrappers/backstage-community-plugin-feedback/dist-dynamic/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "backstage-community-plugin-feedback",
+  "version": "1.6.3",
+  "main": "./dist/index.cjs.js",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "frontend-plugin",
+    "supported-versions": "1.29.2",
+    "pluginId": "backstage-community-plugin-feedback",
+    "pluginPackages": [
+      "backstage-community-plugin-feedback"
+    ]
+  },
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs.js",
+      "default": "./dist/index.cjs.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {},
+  "dependencies": {
+    "@janus-idp/backstage-plugin-feedback": "1.0.1",
```

Comment:
Should this be the MIGRATED plugin in the @community-plugins repo?

Reply:
Your right, I have updated this